### PR TITLE
Remove remaining link checks

### PR DESCRIPTION
**Pull request by davehunt**
We are now using LinkChecker to crawl the site and check all links.

### CODE REVIEW

**Comment by alexgibson** [comment]
LGTM, r+

File: tests/test_partnerships.py [hunk 1]
```diff
@@ -10,26 +10,6 @@
...
-        page = Partnerships(base_url, selenium).open()
-        bad_images = []
-        for image in page.images_list:
-            src = page.image_source(image.get('locator'))
-            if not image.get('img_name_contains') in src:
-                bad_images.append('%s does not contain %s' % (src, image.get('img_name_contains')))
-        assert [] == bad_images
-
-    @pytest.mark.nondestructive
     def test_partner_form_is_visible(self, base_url, selenium):
```
Comment (alexgibson) [inline tests/test_partnerships.py]: I'll make a note to add a test for this in the bedrock repo on our issues list :+1:

Comment (davehunt) [inline tests/test_partnerships.py]: Sure, I'm planning on migrating it (unless you get to it first) to get even closer to removing mcom-tests. :)

**Pull request merged**